frontend/packages/legacy-css/color.scss

```diff
@@ -0,0 +1,252 @@
+// colors from https://www.dropbox.com/s/ycjiwt8vxlmv2ih/Code.org_stylesheet_v2_09.05.13.pdf
```

Comment:
Is there a plan for dashboard to consume this package somehow / keep it in sync with what's in `shared/css/color.scss`? Or is the expectation we'll manage it manually?

Reply:
Yes we will want dashboard to consume this and move `shared/css/color.css` to this version. That being said, @levadadenys would you prefer I keep these files out for now? We'll need them eventually since the components use `color.scss` or is the plan that when we move components to the design system we will also do some work to remove references to `color.scss`?

Reply:
Sorry, I've written a comment, but didn't posted it yet.

I think we won't need it in /frontend since idea is that all the code-dot-org use DSCO Variables (`common/styles/colors.css`, `common/styles/pirimiteColors.css`)

Reply:
Ok -- same question then for other files in this directory like `font.scss`. I don't totally understand how SCSS compilation works in dashboard, but I think these files end up getting dropped into `application.scss` here:

https://github.com/code-dot-org/code-dot-org/blob/c63751fd91bfa3e6d4b77384323aa9019d9fadbe/dashboard/app/assets/stylesheets/application.scss#L34-L37

Would we need to update this pattern if we're getting SCSS from another package? Not sure how much detail to go into here, but basically want to make sure there's a semi-clear path to sharing these files across dashboard and `frontend`.

Reply:
Based on the discussion with @levadadenys the plan is to have a copy of font.scss and fontawesome.scss until we develop a long term solution for font loading (so that we don't block component development). So we won't move fonts just yet and this copy is there to allow us to develop components. I'll add a comment to signify this